docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -0,0 +1,100 @@
+[id="migrating-mgr-from-controller-nodes_{context}"]
+
+= Migrating Ceph Mgr daemons to {Ceph} nodes
+
+The following section describes how to move Ceph Mgr daemons from the
```

Comment:
I'm wondering if all references of "Ceph Mgr" should be "Ceph Manager". It looks that way, from a search on the customer portal, but it seems inconsistent. I can ask Erin if he is back at work on Wednesday, or we'll just merge and fix later.

Reply:
Yeah, it can be named `ceph-mgr` or `Ceph Manager daemon`, but I think that as per [1] I will change it to "Ceph Manager daemon" when it's not a bash command or a code reference.
+1 to follow up with Erin about it!

[1] https://docs.ceph.com/en/latest/mgr/